src/api/authAPI.ts

```diff
@@ -0,0 +1,62 @@
+import type { FetchSignInInfoTypes } from '@/types/authTypes';
+
+const BASE_URL = process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL;
+
+export const getCheckEmailDuplication = async (emailValue: string) => {
+  const url = `${BASE_URL}/api/v1/users/check/email?email=${emailValue}`;
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const getCheckNicknameDuplication = async (nickname: string) => {
+  const url = `${BASE_URL}/api/v1/users/check/nickname?nickname=${nickname}`;
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const postSignup = async (formData: FormData) => {
+  const url = `${BASE_URL}/api/v1/users`;
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        accept: 'application/json',
+      },
+      body: formData,
+    });
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const postSignin = async (formData: FetchSignInInfoTypes) => {
+  const url = `${BASE_URL}/login`;
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+      body: JSON.stringify(formData),
+    });
+    const data = await response.json();
```

Comment:
구조분해해서 data.data로 리턴하는 게 사용할 때 더 쉬울 것 같아요.
 responseData.data.accessToken 이런 식으로 불러오는 것을 responseData.accessToken으로 더 쉽게 불러올 수 있을 것 같아요
```suggestion
    const { data } = await response.json();
```

Reply:
엇 그러면 SignInModal내에서 response의 안의 status하고 Message 확인이 어렵지 않을까요..?!